src/django/api/services/moderation_events_service.py

```
@@ -75,3 +81,17 @@ def handle_processing_error(error_message):
         raise InternalServerErrorException()
+
+    @staticmethod
+    def validate_restriction_role(
+        request,
+        event: ModerationEvent = None
+    ):
+        if request.user.is_superuser:
+            return ModerationEventsService.Role.MODERATOR
+
+        if event:
+            if request.user.contributor.id == event.contributor.id:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)